.github/workflows/release.yml

```
@@ -0,0 +1,105 @@
+        OUTPUT_NAME="digests-api-linux-${{ matrix.arch }}"
+        
+        if [ "${{ matrix.arch }}" = "arm64" ]; then
+          export CC=aarch64-linux-gnu-gcc
+        fi
+        
+        go build -v -ldflags="-s -w" -o ../build/${OUTPUT_NAME} ./cmd/api
+        
+    - name: Upload artifacts
+      uses: actions/upload-artifact@v4
+      with:
+        name: digests-api-${{ matrix.os }}-${{ matrix.arch }}
```

actionlint (1.7.7)

property "os" is not defined in object type {arch: string}

(expression)